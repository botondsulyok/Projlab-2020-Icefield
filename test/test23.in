field spawn0 0
field spawn1 0
field spawn2 0
entity master0 Eskimo spawn0
entity master1 Eskimo spawn1
entity master2 Eskimo spawn2
stats spawn0
stats spawn1
stats spawn2
stats master0
stats master1
stats master2
assemble master0
item pis0 Pistol master0
item cart1 Cartridge master1
item light2 Light master2
stats master0
stats master1
stats master2
assemble master0
assemble master1
assemble master2
item cart0 Cartridge master0
item light1 Light master1
item pis2 Pistol master2
stats master0
stats master1
stats master2
assemble master0
assemble master1
assemble master2
item cart00 Cartridge master0
item light11 Light master1
item pis22 Pistol master2
stats master0
stats master1
stats master2
assemble master0
assemble master1
assemble master2
stats spawn0
stats spawn1
stats spawn2
stats master0
stats master1
stats master2