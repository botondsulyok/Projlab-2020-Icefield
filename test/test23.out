id: spawn0
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master0
pullers: master0
masters: master0
neighbours: null

id: spawn1
type: iceunit
building: plains1
snow: 0
pickable: null
standers: master1
pullers: master1
masters: master1
neighbours: null

id: spawn2
type: iceunit
building: plains2
snow: 0
pickable: null
standers: master2
pullers: master2
masters: master2
neighbours: null

id: master0
type: eskimo
iceUnit: spawn0
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: master1
type: eskimo
iceUnit: spawn1
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: null

id: master2
type: eskimo
iceUnit: spawn2
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: null

id: master0
type: eskimo
iceUnit: spawn0
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis0

id: master1
type: eskimo
iceUnit: spawn1
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: cart1

id: master2
type: eskimo
iceUnit: spawn2
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: light2

id: master0
type: eskimo
iceUnit: spawn0
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis0, cart0

id: master1
type: eskimo
iceUnit: spawn1
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: cart1, light1

id: master2
type: eskimo
iceUnit: spawn2
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: light2, pis2

id: master0
type: eskimo
iceUnit: spawn0
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis0, cart0, cart00

id: master1
type: eskimo
iceUnit: spawn1
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: cart1, light1, light11

id: master2
type: eskimo
iceUnit: spawn2
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: light2, pis2, pis22

id: spawn0
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master0
pullers: master0
masters: master0
neighbours: null

id: spawn1
type: iceunit
building: plains1
snow: 0
pickable: null
standers: master1
pullers: master1
masters: master1
neighbours: null

id: spawn2
type: iceunit
building: plains2
snow: 0
pickable: null
standers: master2
pullers: master2
masters: master2
neighbours: null

id: master0
type: eskimo
iceUnit: spawn0
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pis0, cart0, cart00

id: master1
type: eskimo
iceUnit: spawn1
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: cart1, light1, light11

id: master2
type: eskimo
iceUnit: spawn2
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: light2, pis2, pis22

